app/experimenter/static/rapid/__tests__/experimentDetails.test.tsx

```diff
@@ -36,6 +36,26 @@ describe("<ExperimentDetails />", () => {
     expect(getByDisplayValue("Test objectives")).toBeInTheDocument();
   });
 
+  it("renders without bugzilla info when data provided", async () => {
+    const { getByText } = renderWithRouter(
+      wrapInExperimentProvider(<ExperimentDetails />, {
+        initialState: {
+          slug: "test-slug",
+          name: "Test Name",
+          objectives: "Test objectives",
+          owner: "test@owner.com",
+          features: ["FEATURE 1", "FEATURE 2"],
+          audience: "AUDIENCE 1",
+          bugzilla_url: "https://example.com",
+        },
+      }),
+    );
+
+    await waitFor(() => {
+      return expect(getByText(/Bugzilla ticket/)).toBeInTheDocument();
+    });
```

Comment:
Do we have/need a test where the experiment data does _not_ have a `bugzilla_url` (and we want to assert that `Bugzilla ticket` text is not in the DOM?